Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx

```diff
@@ -245,13 +252,13 @@ void PedestalsCalculationTask::recordPedInDcsCcdb()
   long maxTimeStamp = minTimeStamp + (3600L * mDcsCcdbAliveHours * 1000);
 
   char filename[1024];
-  sprintf(filename, "%s/%s/PedThre.root", mPedestalsCCDBBasePath.c_str(), mPedestalTag.c_str());
-  mExTimer.logMes("File name = >" + std::string(filename) + "< (" + mPedestalsCCDBBasePath + "," + mPedestalTag);
+  sprintf(filename, "%s/%s/PedThre.root", mPedestalsCCDBBasePath.c_str(), PedestalFixedTag.c_str());
+  mExTimer.logMes("File name = >" + std::string(filename) + "< (" + mPedestalsCCDBBasePath + "," + PedestalFixedTag);
```

Comment:
Actually, using the sprintf is dangerous here, you don't know a priory what is the length of the path.
I would use
`auto filename = fmt::format("{}/{}/PedThre.root",mPedestalsCCDBBasePath, PedestalFixedTag);`